lib/computer_player.rb

```diff
@@ -0,0 +1,34 @@
+class ComputerPlayer
+  attr_reader :marker, :board
+
+  def initialize(marker:, board:)
+    @marker = marker
+    @board = board
+    @selections = []
+
+    set_selections
+  end
+
+  def select_space
+    selection = selections.shift
+
+    selection = selections.shift until board.available?(selection)
+
+    selection
+  end
+
+  private
+
+  attr_accessor :selections
+
+  def set_selections
+    srand 1234
```

Comment:
**question:** If we set the random number seed here, is our computer player truly random?

Reply:
There should be a "light bulb" reaction...